stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -245,3 +245,104 @@ where the values specified via the Config API would override values specified as
 | b7a.websub.hub.signaturemethod | "SHA256"      | The signature method to use for authenticated content distribution |
 | b7a.websub.hub.remotepublish   | false         | Whether publishing updates against the topics in the hub could be done by remote publishers via HTTP requests with `hub.mode` set to `publish`  |
 | b7a.websub.hub.topicregistration | true      | Whether a topic needs to be registered at the hub for publishers to publish updates against the topic and for subscribers to send subscription requests for the topic |
+
+## Introducing Specific Subscriber Services (Webhook Callback Services)
+
+Ballerina's WebSub subscriber service listener can be extended to introduce specific Webhooks.
```

Comment:
```suggestion
Ballerina's WebSub-subscriber service listener can be extended to introduce specific Webhooks.
```

Reply:
Not sure if we need the `-` here. Ballerina's WebSub subscriber service listener is the only listener in the WebSub module.

Reply:
+1